packages/core/src/_common/base/login.js

```diff
@@ -33,6 +33,7 @@ const Login = (() => {
         // TODO: [app-link-refactor] - Remove backwards compatibility for `deriv.app`
         const default_login_url_app = `https://oauth.deriv.app/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${website_name.toLowerCase()}`;
         const default_login_url = `https://oauth.deriv.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${website_name.toLowerCase()}`;
+        const default_deriv_crypto_url = `https://oauth.deriv-crypto.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${website_name.toLowerCase()}`;
```

Comment:
can we store this in a variable and reuse it in these three? `?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${website_name.toLowerCase()}`

Reply:
I guess we could add a function to generate the oauth url

Reply:
```suggestion
        const default_deriv_crypto_url = `https://oauth.derivcrypto.com/oauth2/authorize?app_id=${getAppId()}&l=${language}${marketing_queries}&brand=${website_name.toLowerCase()}`;
```